src/page-modules/contact/components/input/input.module.css

```diff
@@ -0,0 +1,39 @@
+@value typo-body__secondary from "@atb/theme/typography.module.css";
+
+.container {
+  composes: typo-body__secondary;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  max-width: 400px;
```

Comment:
I still think the max-width should be decided by the page/component that uses the input-component. This adds more flexibility.